tfx/experimental/pipeline_testing/examples/chicago_taxi_pipeline/taxi_pipeline_regression_test.py

```diff
@@ -0,0 +1,207 @@
+# Lint as: python2, python3
```

Comment:
Can't we reuse existing pipeline definition in https://github.com/tensorflow/tfx/blob/master/tfx/examples/chicago_taxi_pipeline/taxi_pipeline_beam.py ? It looks like most of the content are the same.

Reply:
Can we delete this file if not used any more?